src/collectors/elasticsearch/elasticsearch.py

```diff
@@ -57,6 +57,11 @@ def process_config(self):
                 port = 9200
 
             self.instances[alias] = (host, int(port))
+        
+        if len(self.config['logstash_filter']):
```

Comment:
Won't this always be set since you're defining the default?

This could just be simplified to `re_logstash_filter = re.compile(self.config['logstash_filter'])`